spark/pom.xml

```diff
@@ -41,9 +41,9 @@
 
     <properties>
         <version.org.infinispan>8.5.0-redhat-SNAPSHOT</version.org.infinispan>
-        <version.spark.connector>7.2.0-redhat-SNAPSHOT</version.spark.connector>
+        <version.spark.connector>0.4.0.Final-redhat-2</version.spark.connector>
```

Comment:
I'm not sure about this change... the JIRA just mentions the need for README changes. @gustavonalle Can you verify this is correct?